pkg/utils/source-control-options.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/bugsnag/bugsnag-cli/pkg/log"
+)
+
+func SourceControl(provider string, logger log.Logger) string {
+	if provider == "" {
+		return provider
+	}
+	switch provider {
+	case "github", "github-enterprise", "bitbucket", "bitbucket-server", "gitlab", "gitlab-onpremise":
+		return provider
+	default:
+		logger.Warn(fmt.Sprintf("%s is not an accepted value for the source control provider, proceeding without a provider", provider))
```

Comment:
@tomlongridge Did we want to proceed without this information or did we want to fail and exit ?

Reply:
Fail and exit if they've attempted to provide a value.